docs/development_suite/api-console/api-design/listeners.md

```diff
@@ -91,3 +114,11 @@ Be aware that Endpoints that are **only exposed by** the Listener you are deleti
 :::
 
 ![delete-listener](img/listeners/delete-listener.png)
+
+[//]: # (External links below:)
```

Comment:
what is this notation? 🤔

Reply:
whoops this was to test another way to comment in markdown and i forgot to revert it back